Tools/TestWebKitAPI/Tests/IPC/IPCSerialization.mm

```diff
@@ -853,13 +856,25 @@ static void destroyTempKeychain(SecKeychainRef keychainRef)
     SecTrustRef trustRef = NULL;
     auto policy = adoptCF(SecPolicyCreateBasicX509());
     EXPECT_TRUE(SecTrustCreateWithCertificates(cert.get(), policy.get(), &trustRef) == errSecSuccess);
-    EXPECT_TRUE(trustRef);
+    EXPECT_NOT_NULL(trustRef);
     auto trust = adoptCF(trustRef);
     runTestCF({ trust.get() });
 
     EXPECT_TRUE(SecTrustEvaluateWithError(trust.get(), NULL) == errSecSuccess);
     runTestCF({ trust.get() });
 
+#if HAVE(SEC_ACCESS_CONTROL)
+    NSDictionary *protection = @{(id)kSecUseDataProtectionKeychain : @(YES),
```

Comment:
The style checker doesn't like this bog-standard Objective-C dictionary literal, but I will fight to the death the correctness of this style. 😉